packages/grafana-ui/src/options/builder/axis.tsx

```diff
@@ -96,17 +96,24 @@ export function addAxisConfig(
     });
 
   if (!hideScale) {
-    builder.addCustomEditor<void, ScaleDistributionConfig>({
-      id: 'scaleDistribution',
-      path: 'scaleDistribution',
-      name: 'Scale',
-      category,
-      editor: ScaleDistributionEditor as any,
-      override: ScaleDistributionEditor as any,
-      defaultValue: { type: ScaleDistribution.Linear },
-      shouldApply: (f) => f.type === FieldType.number,
-      process: identityOverrideProcessor,
-    });
+    builder
+      .addCustomEditor<void, ScaleDistributionConfig>({
+        id: 'scaleDistribution',
+        path: 'scaleDistribution',
+        name: 'Scale',
+        category,
+        editor: ScaleDistributionEditor as any,
+        override: ScaleDistributionEditor as any,
+        defaultValue: { type: ScaleDistribution.Linear },
+        shouldApply: (f) => f.type === FieldType.number,
+        process: identityOverrideProcessor,
+      })
+      .addBooleanSwitch({
+        path: 'axisSymmetrical',
+        name: 'Symmetrical scale',
+        category,
+        defaultValue: false,
+      });
```

Comment:
Can u please add a description property to this option? Symmetrical scale is quite vague.  

Does it always align on 0 value?

Reply:
IIUC, it always puts 0 in the middle of the graph -- as a result, different scales will appear aligned to the same location.

As I think about it more (maybe too much!) If the value was a number -1<>+1, this could put the center anywhere, not just the middle of the graph 🤷